GooglePlayInstant/Samples/SphereBlast/scripts/AssetBundleDownloader.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AssetBundleDownloader : MonoBehaviour
+{
+
+	// Use this for initialization
+	public string assetBundleURL;
+
+	IEnumerator Start()
+	{
+		// download and load required scenes
+        Caching.CleanCache();
+        yield return StartCoroutine(
+			DownloadAsset(assetBundleURL, true));
+	}
+
+	IEnumerator DownloadAsset(string sceneURL, bool loadScene)
```

Comment:
Similarly, QuickDeploy provides this as a library function. If we can get that merged this week, we could use it instead.

Reply:
I'll keep as is for now, but will use the QuickDeploy implementation when it's ready